caching.md

```diff
@@ -64,6 +64,11 @@ TBD
 
 TBD
 
+<a name="Caching-%7B%7BOtherCommunitySupportedCacheManagers%7D%7D"></a>
```

Comment:
This anchor isn't needed, Most of the manually defined anchors are for backwards compatibility (the original site was generated/exported through confluence)